llama-index-integrations/tools/llama-index-tools-azure-dynamic-sessions/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Dynamic Sessions Tool
+
+This tool leverages Azure Dynamic Sessions Pool to enable an Agent to run generated Python code in a secure environment with very low latency.
+
+In order to utilize the tool, you will need to have the Session Pool management endpoint first. More details in here: "some-url"
+
+## Usage

```

Comment:
We should include the install statement somewhere, which will be `pip install llama-index-tools-azure-dynamic-sessions` right now